TuneUpTests/config/TestServices.dll.config

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<configuration>
+  <appSettings>
+    <add key="DynamoBasePath" value="C:\Users\t_mitcs\Repos\Dynamo\bin\AnyCPU\Debug"/>
```

Comment:
@ScottAMitchell can we leave this general assuming Dynamo repo is relative to this repo?

Reply:
We actually didn't end up using this, so I'm just removing it.